webpack.config.babel.js

```diff
@@ -137,13 +131,6 @@ export default (env = {}, argv = {}) => {
 				use: [
 					{ loader: 'url-loader' },
 				],
-			}, {
-				test: /\.woff$/,
-				use: [
```

Comment:
Keep this, and just change `conf.target === 'edge' ?
						{ loader: 'url-loader' } :
						{ loader: 'file-loader', options: { name: '[name].[ext]' } }`
to
`{ loader: 'file-loader', options: { name: '[name].[ext]' } }`